types/sector_class.go

```diff
@@ -21,8 +21,8 @@ func NewTestSectorClass() SectorClass {
 // go-filecoin node.
 func NewLiveSectorClass() SectorClass {
 	return SectorClass{
-		postProofPartitions:  OnePoStPartition,
-		poRepProofPartitions: TwoPoRepPartitions,
+		poStProofPartitions:  OnePoStProofPartition,
+		poRepProofPartitions: TwoPoRepProofPartitions,
```

Comment:
The order changed from poRep to poSt to poSt to poRep here. It's a little confusing when comparing Test to Live.

Reply:
Agreed. I've ordered the fields identically for both structs.